allennlp/modules/token_embedders/elmo_token_embedder.py

```diff
@@ -65,11 +65,13 @@ def __init__(self,
                           scalar_mix_parameters=scalar_mix_parameters)
         if projection_dim:
             self._projection = torch.nn.Linear(self._elmo.get_output_dim(), projection_dim)
+            self.output_dim = projection_dim
         else:
             self._projection = None
-
-    def get_output_dim(self):
-        return self._elmo.get_output_dim()
+            self.output_dim = self._elmo.get_output_dim()
+        
```

Comment:
looks like there's some whitespace on this line that pylint doesn't like.

Reply:
Thanks! I should have checked that whitespace problem.

Reply:
@nelson-liu  Thanks for your review and help!
I think i still failed the pylint test. It gives the following error detail:
```
[18:17:00]Using config file /local/deploy/agent2/work/98197cf33cb401e5/.pylintrc
[18:18:48]Process exited with code 1
```
Is there anything i should fix?